canonical/charmhub.io
# Remove filter for draft items

## Done
- Remove filter for draft interfaces
- All interfaces now returned in `/interfaces.json`
## How to QA
- Go to https://charmhub-io-1507.demos.haus/interfaces.json and check that we have both interfaces with `live` and `draft` status
## Issue / Card WD-2150
Fixes #

## Screenshots
